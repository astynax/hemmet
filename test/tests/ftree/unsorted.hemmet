c aa b cc a bb
