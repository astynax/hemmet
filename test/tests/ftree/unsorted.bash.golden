#!/bin/bash
cat <<PREVIEW_END
This file tree will be created:
.
├── a
├── aa
├── b
├── bb
├── c
└── cc
PREVIEW_END
read -p "Press any key to continue..." -n1 -s
set -euf -o pipefail
touch "a"
touch "aa"
touch "b"
touch "bb"
touch "c"
touch "cc"
